### DISCUSSION TITLE
Next Releases: 2.5.0 and 3.0?

### DISCUSSION THREAD

**Discussion by vincentsarago** [stac-utils/stac-fastapi#653] [General]
We've got some fix merged and soon to be merged to `main` branch for which I believe we could do a 2.4.10 or 2.5.0 release. 

There is also a `big` PR which will refactor some part of the code and also make the switch to pydantic 2 #625 which IMO will requite a new `major` version: 3.0

any opinions are welcome

cc @gadomski @jonhealy1

**Comment by jonhealy1** [comment]
A big v3.0.0 push may make sense and then we merge everything we can including #625 and #442. It does also make sense to do a 2.4.10 or 2.5.0 release first, especially if the community is really wanting some of these other changes available immediately. Basically IDK.

**Comment by vincentsarago** [comment]
🥳 https://github.com/stac-utils/stac-fastapi/releases/tag/2.5.0

**Comment by jonhealy1** [comment]
@vincentsarago What do we want to get finished for the v3.0.0 release?

**Reply by vincentsarago** [reply]
I'm not quite sure yet, my plan was to try to move some production stack (e.g Planetary computer) to 3.0.0 alpha and see if we encounter any issue.

**Comment by vincentsarago** [comment]
I think we can start planning the official release of 3.0 after we merge the last PR #739 

### To Do 

- [x] remove deprecation 
- [x] update submodule pinning (#678)

**Comment by vincentsarago** [comment]
🥳 https://github.com/stac-utils/stac-fastapi/releases/tag/3.0.0

Thanks everyone who contributed to this 🙏 @jonhealy1 @thomas-maschler

**Comment by vprivat-ads** [comment]
The milestones should be consistent with releases, currently they both include open tickets: https://github.com/stac-utils/stac-fastapi/milestones

**Reply by vincentsarago** [reply]
I think some of those tickets can be closed, the rest should be moved to 4.0.0
